### PR TITLE
Add support for reversed sorting of comments

### PR DESCRIPTION
**Pull request by odinuge**
The prop 'newOnTop' for CommentView allows comments to be sorted (only
the first level in the tree) with the newest comments on top. The
comment form will then also be on the top.

screenshot from 2017-10-27 01-01-10

### CODE REVIEW

**Comment by edvardvb** [comment]
This does not sort replies, only the top-level comments, right?

File: app/components/Comments/CommentView.js [hunk 1]
```diff
@@ -26,28 +31,38 @@ const CommentView = (props: Props) => {
...
+      <Title displayTitle={displayTitle} />
+      <Flex
+        style={{
+          flexDirection: newOnTop ? 'column-reverse' : 'column'
+        }}
+      >
+        <LoadingIndicator loading={!comments}>
+          {comments && (
+            <CommentTree
+              comments={newOnTop ? tree.reverse() : tree}
```
Comment (odinuge) [inline app/components/Comments/CommentView.js]: Yeah @edvardvb. It reverses the first layer of comments in the generated tree, not the rest of them.

**Comment by ekmartin** [comment]
Couldn't we just always return comments sorted like this from the backend? Newest first seems reasonable to me - i.e. do we need to support both options?

**Comment by finnss** [comment]
We can always remove support for oldest first later. I'll get this merged so we have the functionality for sorting by newest first at all.

**Pull request merged**